### PR TITLE
Updated membercount checks

### PR DESCRIPTION
**Pull request by TheHollidayInn**

### CODE REVIEW

File: website/server/libs/amazonPayments.js [hunk 1]
```diff
@@ -270,10 +270,8 @@ api.subscribe = async function subscribe (options) {
   let priceOfSingleMember = 3;
 
   if (groupId) {
-    let groupFields = basicGroupFields.concat(' purchased');
-    let group = await Group.getGroup({user, groupId, populateLeader: false, groupFields});
-
-    amount = sub.price + (group.memberCount - leaderCount) * priceOfSingleMember;
+    const membersCount = await User.count({guilds: this._id}).exec();
```
Comment (paglias) [inline website/server/libs/amazonPayments.js]: What if the group is a party?

Comment (TheHollidayInn) [reply website/server/libs/amazonPayments.js]: Good point :P

Comment (paglias) [reply website/server/libs/amazonPayments.js]: might be a good idea to extract this check in a reusable function, in particular if it becomes more complex when you have to distinguish between parties and guilds

Comment (TheHollidayInn) [reply website/server/libs/amazonPayments.js]: Yea, I was planning on doing that after you suggested the party

**Pull request merged**